components/collection/HeroButtons.vue

```diff
@@ -0,0 +1,169 @@
+          <NeoButton icon="ellipsis-vertical" class="square-32" />
+          <template #items>
+            <div v-if="isOwner">
+              <NeoDropdownItem>
+                {{ $i18n.t('moreActions.delete') }}
+              </NeoDropdownItem>
+              <NeoDropdownItem>
+                {{ $i18n.t('moreActions.customize') }}
+              </NeoDropdownItem>
+            </div>
+            <div v-else>
+              <NeoDropdownItem>
+                {{ $i18n.t('moreActions.reportNFT') }}
+              </NeoDropdownItem>
+              <NeoDropdownItem>
+                <div class="is-flex">
+                  {{ $i18n.t('moreActions.download') }}
+                  <b-icon icon="download" class="pl-3" />
+                </div>
+              </NeoDropdownItem>
+            </div>
+          </template>
+        </NeoDropdown>
+      </div>
+    </div>
+    <b-modal v-model="QRModalActive">
+      <div class="card">
+        <header class="card-header">
+          <p class="card-header-title">{{ collection?.name }}</p>
+        </header>
+        <div class="card-content">
+          <QRCode :text="currentURL" color="#db2980" bg-color="#000" />
+        </div>
+      </div>
+    </b-modal>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoDropdown, NeoDropdownItem } from '@kodadot1/brick'
+import { isOwner as checkOwner } from '@/utils/account'
+import { useCollectionMinimal } from './utils/useCollectionDetails'
+import useIdentity from '@/components/identity/utils/useIdentity'
+
+const route = useRoute()
+const { accountId } = useAuth()
+const { $i18n, $buefy } = useNuxtApp()
+const collectionId = computed(() => route.params.id)
+const currentURL = computed(() => window.location.href)
+const { collection } = useCollectionMinimal({
+  collectionId: collectionId.value,
+})
+const collectionIssuer = computed(() => collection.value?.issuer)
+
+const { discord, twitter, instagram, whichIdentity } = useIdentity({
+  address: collectionIssuer.value,
+})
+
+watch(collectionIssuer, () => {
+  whichIdentity(collectionIssuer.value)
+})
+
+const displaySeperator = computed(
+  () => discord.value || twitter.value || instagram.value
+)
+const isOwner = computed(() =>
+  checkOwner(collection.value?.currentOwner, accountId.value)
+)
+
+const QRModalActive = ref(false)
+
+const hashtags = 'KusamaNetwork,KodaDot'
+const sharingLabel = $i18n.t('sharing.collection')
+
+const toast = (message: string) => {
+  $buefy.toast.open({
+    message,
+    type: 'is-neo',
+  })
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+.hero-buttons {
+  @include mobile {
+    justify-content: space-between !important;
+    flex: 1;
+    margin-top: 0;
+    margin-bottom: 1.5rem;
+  }
+}
+.square-32 {
+  width: 32px;
+  height: 32px;
+}
+.vertical-seperator {
+  height: 24px;
+  margin-bottom: 4px;
+  width: 1px;
+  @include ktheme() {
+    background-color: theme('border-color');
+  }
```

Comment:
can we avoid pxs?

Reply:
for square-32 I don't see problem to use px, but can't we reuse same buttons component as the one in footer?

Reply:
I just take a look at the components in footer and found that there are lots of custom style and tag like `<li> <a> <b-icon> ` : )  We use `NeoButton` here would be a better choice.
<img width="754" alt="image" src="https://user-images.githubusercontent.com/31397967/226927427-b47003e9-bb15-4c3c-adcc-8cce6410ddfe.png">

Reply:
Aight leave it like this for now